api/exporter.go

```diff
@@ -84,6 +86,13 @@ func (s *StorageExporterService) Export(r *http.Request, args *StorageExporterAr
 	ctx := context.TODO()
 	result, err := exp.Export(ctx, core.PulseNumber(args.From), args.Size)
 	if err != nil {
+		if strings.Contains(err.Error(), "failed to fetch pulse data") {
+			return &jsonrpc.Error{
+				Code:    10404,
```

Comment:
Shouldn't we extract error code to constant?

Reply:
No